recipes/libs/compat/glibc.yaml

```diff
@@ -36,6 +36,7 @@ buildScript: |
             --with-headers=$2/usr/include \
             --enable-obsolete-rpc \
             --libdir=/usr/lib \
+            --disable-werror \
```

Comment:
Why is this needed? I'd rather add a patch than disabling werror altogether...

Reply:
while cross-compiling for aarch64 i had an error: `-Werror=missing-attributes`
with `--disable-werror` i was able to build it successfully